docs/api/api_blueprint.apib

```diff
@@ -20828,6 +20828,90 @@ Get the task status and the final result of the task as response
         }
 
 
+## Event Types Statistics Details [/v1/admin/statistics/event-types]
+
+**Event Types:**
+
+| Parameter | Description | Type |
```

Comment:
Do these parameters types match the response ? Can you post a sample response here ?

Reply:
@niranjan94 
For **EventTypes**`/v1/admin/statistics/event-types`

![screenshot from 2018-06-09 15-15-49](https://user-images.githubusercontent.com/22936570/41190212-309314d4-6bf8-11e8-8bfd-0c484931336b.png)

For **EventTopics**`/v1/admin/statistics/event-topics` 
	
![screenshot from 2018-06-09 15-15-35](https://user-images.githubusercontent.com/22936570/41190208-102bdac8-6bf8-11e8-9bb3-9b0c343fabaf.png)

Reply:
@bhaveshAn please make the response something like,

```js
[
    {
        name: "Appearance or Signing",
        id: 1,
        count: 12
    },
    {
        name: "Attraction",
        id: 2,
        count: 1
    }
]
```

That way the clients have access to the ID of the type too. This would allow clients to implement features such as viewing all events of a specific type right from the statistics page

Reply:
@niranjan94 Done, Please review.

**EventTypes**
![screenshot from 2018-06-09 18-25-23](https://user-images.githubusercontent.com/22936570/41191806-8cdd3f7a-6c12-11e8-9b32-b7c1f1525932.png)
**EventTopics**
![screenshot from 2018-06-09 18-25-15](https://user-images.githubusercontent.com/22936570/41191807-8d674b52-6c12-11e8-8cc1-784bcbb87812.png)